index.bs

```diff
@@ -1,5 +1,9 @@
 <pre class="metadata">
 Title: Controlled Frame API
+Abstract: This document defines an API for embedding arbitrary web content only
+    within the context of a Isolated Web Applications (IWAs). The embedded
+    content is a new top-level browsing context that is able to be controlled by
```

Comment:
How about here: "top-level browsing context within and controlled by the embedder." (An implementation's API is assumed to have control over the implementation I believe in most cases.)